qdrant-landing/content/articles/qdrant-1.9.x.md

```diff
@@ -0,0 +1,88 @@
+draft: false
+slug: qdrant-1.9.x 
+short_description: "Granular access control. Optimized shard transfers. Support for byte embeddings."
+description: "New access control options for RBAC, a much faster shard transfer procedure, and direct support for byte embeddings. " 
+social_preview_image: /articles_data/qdrant-1.9.x/social_preview.png
+small_preview_image: /articles_data/qdrant-1.9.x/icon.svg
+preview_dir: /articles_data/qdrant-1.9.x/preview
+weight: -150
+date: 2024-04-24T00:00:00-08:00
+author: David Myriel
+featured: false 
+tags:
+  - vector search
+  - role based access control
+  - byte vectors
+  - binary vectors
+  - quantization
+  - new features
+---
+
+[Qdrant 1.9.0 is out!](https://github.com/qdrant/qdrant/releases/tag/v1.9.0) This version complements the release of our new managed product [Qdrant Hybrid Cloud](https://hybrid-cloud.qdrant.tech/) with key security features valuable to our enterprise customers, and all those looking to productionize large-scale Generative AI. **Data privacy, system stability and resource optimizations** are always on our mind - so let's see what's new:
+
+- **Granular access control:** You can further specify access control levels by using JSON Web Tokens. 
+- **Optimized shard transfers:** The transfer of shards between nodes is now significantly faster!
+- **Support for byte embeddings:** Reduce the memory footprint of Qdrant with official `uint8` support.
+
+## New access control options via JSON Web Tokens
+
+Historically, our API key supported basic read and write operations. However, recognizing the evolving needs of our user base, especially large organizations, we've implemented additional options for finer control over data access within internal environments.
+
+Qdrant now supports [granular access control using JSON Web Tokens (JWT)](/documentation/guides/security/#granular-access-control-with-jwt). JWT will let you easily limit a user's access to the specific data they are permitted to view. Specifically, JWT-based authentication leverages tokens with restricted access to designated data segments, laying the foundation for implementing role-based access control (RBAC) on top of it. **You will be able to define permissions for users and restrict access to sensitive endpoints.**
+
+> **Documentation:** [Read the access level breakdown](/documentation/guides/security/#table-of-access) to see which actions are allowed or denied.
+
+We highly recommend this for enterprises using [Qdrant Hybrid Cloud](https://hybrid-cloud.qdrant.tech/), as it is tailored to those who need additional control over company data and user access. RBAC empowers administrators to define roles and assign specific privileges to users based on their roles within the organization. In combination with [Hybrid Cloud's data sovereign architecture](/documentation/hybrid-cloud/), this feature reinforces internal security and efficient collaboration by granting access only to relevant resources.
```

Comment:
It is not supported in (hybrid) cloud right away. This probably lands a few weeks later.

In short, it is disabled by default and users need to set a configuration properly to make use of it. In cloud this cannot be done, but they'll support it through a different channel later.

We can probably leave it in, but might have to add some nuance here.